javascript/4-intro-to-dom/hw-4/scripts.js

```diff
@@ -1 +1,7 @@
-// Your JS code is here
+function countTags(tag) {
+  var numberOfTags = document.querySelectorAll(tag);
```

Comment:
// you could do this way too
var numberOfTags = document.querySelectorAll(tag).length;
console.log(`There are ${numberOfTags}  tags of type ${tag} on the page`);
// so the variable is number already, but both ways are fine, good job!